source/server/http_dynamic_delay_filter.h

```diff
@@ -66,10 +66,18 @@ class HttpDynamicDelayDecoderFilterConfig : public FilterConfigurationBase {
    */
   Envoy::Stats::Scope& scope() { return scope_; }
 
+  /**
+   * @return Envoy::Server::Configuration::ServerFactoryContext& to be used by filter
+   * instantiations associated to this.
+   */
+  Envoy::Server::Configuration::ServerFactoryContext& server_factory_context() {
+    return server_factory_context_;
+  }
+
   /**
    * @return Envoy::TimeSource& to be used by filter instantiations associated to this.
    */
-  Envoy::TimeSource& time_source() { return time_source_; }
+  // Envoy::TimeSource& time_source() { return server_factory_context_.timeSource(); }
```

Comment:
Did you mean to leave this commented out?

Reply:
Now deleted this method since it's no longer called.